presto-raptor/src/test/java/com/facebook/presto/raptor/integration/TestRaptorIntegrationSmokeTest.java

```diff
@@ -795,4 +795,15 @@ public void testDelete()
 
         assertUpdate("DROP TABLE test_delete_table");
     }
+
+    @Test
+    public void testTriggerBucketBalancer()
+    {
+        assertUpdate("CREATE TABLE test_bucket_balancer (c1 bigint, c2 bigint)");
```

Comment:
I think we don't need to create a table for this test. Just call the procedure and see if it fails.